packages/fx-core/resource/package.nls.json

```diff
@@ -529,6 +529,33 @@
   "core.copilot.addAPI.success": "%s have(has) been successfully added to %s",
   "core.copilot.addAPI.InjectAPIKeyActionFailed": "Inject API key action to teamsapp.yaml file unsuccessful, make sure the file contains teamsApp/create action in provision section.",
   "core.copilot.addAPI.InjectOAuthActionFailed": "Inject OAuth action to teamsapp.yaml file unsuccessful, make sure the file contains teamsApp/create action in provision section.",
+  "core.uninstall.botNotFound": "Cannot find bot using the manifest ID %s",
+  "core.uninstall.confirm.tdp": "App registration of manifest ID: %s will be uninstalled. Please confirm.",
+  "core.uninstall.confirm.m365App": "Microsoft 365 Application of Title ID: %s will be uninstalled. Please confirm.",
+  "core.uninstall.confirm.bot": "Bot framework registration of bot ID: %s will be uninstalled. Please confirm.",
```

Comment:
Proposed: Bot framework registration of bot ID: %s will be canceled. Please confirm.